src/workerd/util/sqlite.c++

```diff
@@ -437,9 +445,10 @@ bool SqliteDatabase::isAuthorized(int actionCode,
   }
 
   KJ_IF_MAYBE(d, dbName) {
-    if (*d != "main"_kj) {
-      // We don't allow opening multiple databases (including temporary databases), as our storage
-      // engine is not designed for this at present.
+    if (*d != "main"_kj && *d != "temp"_kj) {
```

Comment:
Can we possibly be a bit more cautious and allow "temp" for `READ` operations only? That looks like it may solve the `ALTER TABLE` problem (weirdly enough). I don't want to fully enable temp databases until we have implemented them in a way that doesn't just put everything in RAM in the edge runtime.